main.nf

```diff
@@ -361,7 +361,8 @@ process fastqc {
  */
 process trim_galore {
     tag "$name"
-    publishDir "${params.outdir}/trim_galore", mode: 'copy'
+    publishDir "${params.outdir}/trim_galore", mode: 'copy',
```

Comment:
If the filename contains the string `_fastqc` (the index position of that substring is > 0), the filter is applied which prepends `FastQC/` to the filename.

In other words, it puts the FastQC reports into a subdirectory.